charger/easee.go

```diff
@@ -280,6 +281,50 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// received observation is outdated, ignoring
 		return
 	}
+
+	switch res.ID {
+		case easee.CHARGER_OP_MODE:
+			//for every op mode change, if last energy updated was triggered more than 3 minutes ago, request new update
```

Comment:
Do we need this for every opMode change? Can't we apply logic similar toL299 below, e.g. only trigger poll if new opMode is 2 (charging) or 1 (disconnected)?

Reply:
Yes we could limit to some op modes, but does it hurt to update more frequently, obeying the limit of max 1 call in 3 minutes?
But we could.
With authentication, op mode 7 (awaiting authentication) is one op mode to use as trigger.
No matter if the charging will start immediately or later (depending on the evcc charging mode), we will receive an actual lifetime energy shortly, which will not change until evcc or the user starts the charging.

Also, if EVCC stops the charging (in op mode 3), op mode 7 will be the next op mode to reach.
But if the user stops the charging by unlocking and disconnecting the car, the op mode will change from 3 to 1. 
So **reaching** op mode 1 or 7 should be sufficient, no matter of the previous op mode.
This will update also if evcc pauses the charging (op mode 3->7) - and update the UI, what's ok.

But i'm not sure about boxes without authentication, which op mode will follow on which op mode.
I assume connecting a car will change from 1 to (6 or 2) to 3. Not sure. But no matter what, we could use **leaving** op mode 1, no matter of the next op mode.  Again if the user stops the charging, the op mode should change from 3 to 1, so **leaving** op mode 3 could trigger.

Sumamry, reaching 1 and 7, or leaving 1 and 3, should trigger. What do you think?